### PR TITLE
add function for getting tokenId from contractId and tokenIndex

### PR DESCRIPTION
**Pull request by PKUzjw**

### CODE REVIEW

File: src/utils/common.js [hunk 1]
```diff
@@ -71,5 +71,17 @@ export default {
         let contractArr = encodeArr.concat(checkArr)
         let contractString = Base58.encode(contractArr)
         return contractString
+    },
+    getTokenIdByIndex(contraId, tokenIndex) {
```
Comment (SheldonYS) [inline src/utils/common.js:75]: This function is similar with contractIDToTokenID. Just change that function to `contractIDToTokenID(contractId, tokenIndex=0)` should be OK.

Comment (PKUzjw) [reply src/utils/common.js:75]: ok

**Comment by SheldonYS** [comment]
Please open get token ID function by Contract ID and Token Index for public using (change in `utils/convert.js`)

File: src/utils/common.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ export default {
         let index = Converters.byteArrayToSignedInt32(indexBytes)
         return index
     },
-    contractIDToTokenID(contraId) {
+    contractIDToTokenID(contraId, tokenIndex=0) {
         let conId = Base58.decode(contraId)
         let firstArr = [132]
         let secondArr = Array.from(conId.slice(1,conId.length-4))
```
Comment (SheldonYS) [inline src/utils/common.js:55]: change "0"  in Line 56 to `tokenIndex`.

**Review by SheldonYS — APPROVED** [review]

**Pull request merged**